fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadMgr.java

```diff
@@ -236,15 +242,16 @@ public void addRoutineLoadJob(RoutineLoadJob routineLoadJob, String dbName) thro
                         + dbName);
             }
             long allSlotNum = beNum * Config.max_routine_load_task_num_per_be;
+            // When calculating the used slots, we only consider Jobs in the NEED_SCHEDULE and RUNNING states.
             List<RoutineLoadJob> jobs = getRoutineLoadJobByState(Sets.newHashSet(RoutineLoadJob.JobState.NEED_SCHEDULE,
-                        RoutineLoadJob.JobState.RUNNING, RoutineLoadJob.JobState.PAUSED));
-            long curSlotNum = 0;
+                        RoutineLoadJob.JobState.RUNNING));
+            long curTaskNum = 0;
             for (RoutineLoadJob job : jobs) {
-                curSlotNum += job.calculateCurrentConcurrentTaskNum();
+                curTaskNum += job.calculateCurrentConcurrentTaskNum();
```

Comment:
This looks even more confusing, the same function returns values with two different names.

Reply:
> This looks even more confusing, the same function returns values with two different names.

@meegoo updated